### PR TITLE
Rename webpack

### LINKED ISSUE #594: Update from gulp-webpack to webpack-stream
gulp-webpack is deprecated...

https://www.npmjs.com/package/webpack-stream

### PR DESCRIPTION
**Pull request by zckrs**
close #594

### CODE REVIEW

**Comment by Swiip** [comment]
:+1:

File: app/templates/gulp/_scripts.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ var gulp = require('gulp');
 var conf = require('./conf');
 
 var browserSync = require('browser-sync');
+var webpack = require('webpack-stream');
```
Comment (zckrs) [inline app/templates/gulp/_scripts.js]: @Swiip ... I forgot to add the condition on this require :-(

Comment (Swiip) [inline app/templates/gulp/_scripts.js]: arf :)

Comment (zckrs) [inline app/templates/gulp/_scripts.js]: https://github.com/Swiip/generator-gulp-angular/pull/648

**Pull request merged**